log_outgoing_requests/admin.py

```diff
@@ -100,6 +103,10 @@ def request_body(self, obj) -> str:
     def response_body(self, obj) -> str:
         return obj.response_body_decoded or "-"
 
+    @admin.display(description=_("Response content length"))
+    def response_content_length(self, obj):
+        return obj.response_content_length
```

Comment:
I think this can be removed, adding the field to `readonly_fields` should be enough to display it in the admin